stdlib/LinearAlgebra/test/dense.jl

```diff
@@ -220,9 +191,41 @@ end
         @test Matrix(factorize(A)) ≈ Matrix(factorize(Tridiagonal(e2,d,e)))
         A = diagm(0 => d, 1 => e, 2 => f)
         @test factorize(A) == UpperTriangular(A)
+
+        x = rand(eltya)
+        @test factorize(x) ≈ x
     end
 end # for eltya
```

Comment:
The loop over `eltya` ends here, and - without change to the code - I moved the existing three test suites below.